repos/c2sm/packages/icon/package.py

```diff
@@ -193,6 +193,10 @@ class Icon(AutotoolsPackage, CudaPackage):
         depends_on('py-gridtools-cpp', when='dsl={0}'.format(x))
         conflicts('^python@:3.9,3.11:', when='dsl={0}'.format(x))
 
+    depends_on('autoconf', type='build', when='@2.6.6')
+    depends_on('automake', type='build', when='@2.6.6')
+    depends_on('libtool', type='build', when='@2.6.6')
+
```

Comment:
Can you clarify why this dependencies are only needed for `@2.6.6` but not for other versions?